### PR TITLE
Move sofb 2 dev

### PR DESCRIPTION
**Pull request by fernandohds564**
merge only after #408 is merged.

### CODE REVIEW

File: siriuspy/siriuspy/sofb/correctors.py [hunk 1]
```diff
@@ -0,0 +1,573 @@
+"""Module to deal with correctors."""
+
+import time as _time
+import logging as _log
+import numpy as _np
+from .. import util as _util
```
Comment (xresende) [inline siriuspy/siriuspy/sofb/correctors.py]: I am adopting the standard of separating groups of import by empty line. groups are: 1) default python packages, 2) sirius-developed packages other than current one, 3) imports ..., 4) imports .., and 5) imports .

File: siriuspy/siriuspy/sofb/main.py [hunk 2]
```diff
@@ -0,0 +1,593 @@
+"""Main Module of the program."""
+
+import time as _time
+import logging as _log
+from functools import partial as _part
+from threading import Thread as _Thread
+import numpy as _np
+
+from ..epics import PV as _PV
```
Comment (xresende) [inline siriuspy/siriuspy/sofb/main.py]: same comment. missing one empty separation line here, according to the standard I am using. you guys agree?

File: siriuspy/siriuspy/sofb/matrix.py [hunk 3]
```diff
@@ -0,0 +1,336 @@
+"""Class of the Response Matrix."""
+
+import os as _os
+from copy import deepcopy as _dcopy
+import logging as _log
+import numpy as _np
+from functools import partial as _part
+from .base_class import BaseClass as _BaseClass
```
Comment (xresende) [inline siriuspy/siriuspy/sofb/matrix.py]: same

File: siriuspy/siriuspy/sofb/orbit.py [hunk 4]
```diff
@@ -0,0 +1,900 @@
...
+import logging as _log
+from functools import partial as _part
+from threading import Lock, Thread
+from copy import deepcopy as _dcopy
+import numpy as _np
+
+from .. import util as _util
+from ..csdevice import bpms as _csbpm
+from ..thread import RepeaterThread as _Repeat
+from .base_class import BaseClass as _BaseClass
```
Comment (xresende) [inline siriuspy/siriuspy/sofb/orbit.py]: same

File: siriuspy/siriuspy/sofb/correctors.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/sofb/correctors.py]: Ok, I agree. Can I fix this in next PR. I'm already preparing one.

**Pull request merged**